bluebase/storybook-native/storybook/storyLoader.js

```diff
@@ -0,0 +1,22 @@
+
+// Auto-generated file created by react-native-storybook-loader
```

Comment:
add this file to gitignore, does not need to be committed.

Reply:
Done

Reply:
Done